docs/actions.md

```diff
@@ -567,16 +570,16 @@ The action may be invoked as any other OpenWhisk action.
   }
   ```
 
-If you update the Docker image, run `buildAndPush.sh` to refresh the image on Docker Hub, then run `wsk action update` to make the system to fetch the new image. New invocations will start using the new image and not a warm image with the old code.
-
+  To update the Docker image, run `buildAndPush.sh` to refresh the image on Docker Hub, then run `wsk action update` to make the system to fetch the new image. New invocations will start using the new image and not a warm image with the old code.
```

Comment:
This may be interpreted to mean that in the absence of `update` the system uses the old image. That's not the case if for example the image uses the `latest` tag. Every cold invoke will pull the latest tag even if action is not updated.